app/config.py

```diff
@@ -3,7 +3,7 @@
 
 from dotenv import load_dotenv
 
-SHA1 = subprocess.getoutput("git rev-parse HEAD")
+SHA1 = lambda: subprocess.getoutput("git rev-parse HEAD")
```

Comment:
Could you explain why `SHA1` should be a function here?

Reply:
subprocess module spawn a new os-level process, the config module is interpreted at every http request, so it a waste of cpu

Reply:
Ah actually the `SHA1` is evaluated only once first time when `config` module is imported. You can test that by adding this snippet of toy-code to `config.py`: the `>>>b is called` should be printed only once.

```
def b():
    print(">>>b is called")
    return 10

A = b()
```

Reply:
You're right, I was so stupid to think the every variable is garbage collected after a request. You may now close this PR since it's built upon my misunderstanding on how Wezeurg works.

Reply:
No worry :), this is not clearly said in Flask doc neither. I'll close this PR but will make sure to use update the doc to use Python 3.7+.